eng/pipelines/docs.yml

```diff
@@ -7,75 +7,103 @@ jobs:
     pool:
       vmImage: vs2017-win2016
     steps:
+      - pwsh: |
+          Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://github.com/mono/api-doc-tools/releases/download/mdoc-5.7.4.9/mdoc-5.7.4.9.zip" `
+          -OutFile "mdoc.zip" | Wait-Process; Expand-Archive -Path "mdoc.zip" -DestinationPath "./mdoc/"
+        workingDirectory: $(Build.BinariesDirectory)
+        displayName: Download and Extract mdoc Zip
       - pwsh: |
           Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
           -OutFile "docfx.zip" | Wait-Process; Expand-Archive -Path "docfx.zip" -DestinationPath "./docfx/"
         workingDirectory: $(Build.BinariesDirectory)
         displayName: Download and Extract DocFX
+      - pwsh: |
+          dotnet restore "$(Build.SourcesDirectory)/eng/docgeneration/assets/docgen.csproj" --packages docgenpackages
```

Comment:
I don't think you want to be passing in the `--packages` property to these restore commands any longer.

Reply:
Removed